main.js

```diff
@@ -605,7 +607,7 @@ function updatePersistentDataForm(iniobj) {
         document.getElementById("ini-dodged-all-special-thanks").checked = false;
     }
     if (iniobj.fun) {
-        document.getElementById("ini-fun").value = parseInt(iniobj.fun.trim());
+        //document.getElementById("ini-fun").value = parseInt(iniobj.fun.trim());
```

Comment:
What happened to this? Why is it commented out? Is there a replacement functionality?

Reply:
I tried out the system several times and I didn't see its value change neither in our project nor in your project. Same thing with "sav-kills" value. Also I needed some space to be able to make labels as readable as possible. I can add them back if they are needed for the system :)